website/docs/cdktf/examples.html.md

```diff
@@ -71,8 +71,23 @@ Follow these hands-on tutorials from HashiCorp Learn: [Write CDK for Terraform C
 
 ## Youtube Playlist
 
-TODO Add information here with a link.
+* Watch the CDK for Terraform engineering team build a simple end to end serverless application in an [end to end livestream](https://www.youtube.com/watch?v=Ey0SW0c6p8c)
+* [Getting Started with CDK for Terraform and Python](https://www.youtube.com/watch?v=Ee2qh-pEC5k&t=258s), by Charles mcLaughlin of ShopStyle
+* [Extending constructs of the CDK for Terraform](https://www.youtube.com/watch?v=cfU-WOGdNqA)
 
-## Pocket public codebase?
+### Release demos
+[0.4: Go support, Asset construct, Terraform Cloud integration](https://www.youtube.com/watch?v=TTfFAIeSqgo). Includes a [GKE demo](https://youtu.be/TTfFAIeSqgo?t=1573).
+[0.5: Convert and watch command demos](https://www.youtube.com/watch?v=4caW8WJM4h4&t=1s)
+[0.6: snapshot testing](https://www.youtube.com/watch?v=9Is4QJT2664)
 
-TODO Describe what this is and add a link :-)
+Constributors interested in extending the `cdktf convert` command may be interested in the [convert deep dive](https://www.youtube.com/watch?v=rSn4-Ki5nho). This is not needed to use it.
+
+## Pocket public codebase
+
+Mozilla Pocket is a widely used application for managing reading lists that's built into the Firefox browser. Like many Mozilla projects, Pocket is open source, and the CDK for Terraform codebase that they use to manage infrastructure for their recommendation API is also public and open source. Pocket's codebase provides a great example of how to lay out a CDK for Terraform project.
+
+In order to re-use components, Pocket's codebase is separated out into a [set of reusable modules](https://github.com/Pocket/terraform-modules/tree/main/src/pocket). These are then used from CDK code in the [recommendation-api codebase](https://github.com/Pocket/recommendation-api/tree/main/.aws). The recommended reading order is to:
```

Comment:
Did you mean from CDK for Terraform code? Just checking!

Reply:
Yes! If you can draft example-summaries that would be a great improvement